Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -471,6 +479,45 @@ public extension UIView {
         CATransaction.commit()
     }
 
+    /// SwifterSwift: Apply Gradient Colors.
```

Comment:
Let's keep this PR only for documentation changes and move those new extensions into separated one. Normally we try one per extension =]

Reply:
Sorry, I can't wait to submit a few great extensions.